numba/np/arrayobj.py

```diff
@@ -3976,7 +3978,9 @@ def iternext_specific(self, context, builder, arrty, arr, result):
                                                                  indices,
                                                                  dim))
                                for dim in range(ndim)]
-                    idxtuple = cgutils.pack_array(builder, idxvals)
+                    idxtuple = cgutils.pack_array(builder, idxvals,
```

Comment:
The question here is: when is a zero-dimensional arry non-contigous, and does the test-case cover that?